src/dispatch/static/dispatch/src/incident/TableExportDialog.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <v-dialog v-model="showExport" persistent max-width="800px">
+    <template v-slot:activator="{ on }">
+      <v-btn color="secondary" class="ml-2" dark v-on="on">Export</v-btn>
+    </template>
+    <v-card>
+      <v-card-title>
+        <span class="headline">Export Incidents</span>
+      </v-card-title>
+      <v-stepper v-model="e1">
+        <v-stepper-header>
+          <v-stepper-step :complete="e1 > 1" step="1" editable>
+            Filter Data
+          </v-stepper-step>
+          <v-divider></v-divider>
+
+          <v-stepper-step :complete="e1 > 2" step="2" editable>
+            Select Fields
+          </v-stepper-step>
+
+          <v-divider></v-divider>
+
+          <v-stepper-step step="3" editable>
+            Preview
+          </v-stepper-step>
+        </v-stepper-header>
+
+        <v-stepper-items>
+          <v-stepper-content step="1">
+            <v-list dense>
+              <v-list-item>
+                <v-list-item-content>
+                  <tag-filter-combobox v-model="filters.tag" label="Tags" />
+                </v-list-item-content>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-content>
+                  <incident-type-combobox v-model="filters.incident_type" />
+                </v-list-item-content>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-content>
+                  <incident-priority-combobox v-model="filters.incident_priority" />
+                </v-list-item-content>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-content>
+                  <incident-status-multi-select v-model="filters.status" />
+                </v-list-item-content>
+              </v-list-item>
```

Comment:
Are we missing a label for these three?

Reply:
I believe they all have default labels within the components themselves.